android/app/src/main/java/co/bitnation/PanthalassaModule.java

```diff
@@ -475,6 +519,8 @@ public void run() {
     }
 
     //=====
+
+    // This method should be deleted due is not the active protocol listener now
```

Comment:
If that method is not required to be kept, let's remove it right now.

Reply:
Done